### ISSUE TITLE
Allow Softmax Option 

### ISSUE THREAD

**Issue by jlevy44** [shap/shap#587]
Hey there!

I think I recently ran into this problem. I'd like to explain my pytorch model, which is a 30-class classifier, but it has a linear layer as its output layer (no transformation at the end), and a separate loss function computes log-softmax and NLLLoss to constitute the cross entropy loss. 

I wonder if not Softmaxing my outputs will cause issues when interpreting this model. Could you add a softmax option perhaps for the GradientExplainer? That way, my SHAP values have more interpretability.

**Comment by slundberg** [comment]
I don't use PyTorch regularly, but can't you just define a new model that is the softmax transform of the original one and then use that?

**Comment by jlevy44** [comment]
Yeah I can do that. I had just trained my model and was looking for a lazy way forward. I’ll just have to use the state dict to transfer the weights. Thanks!
